tests/lib/bin/lxd-tool

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -e
```

Comment:
I like this, but I have one question for you ... why not python? :stuck_out_tongue_winking_eye:

Reply:
Challenge accepted!